### PR TITLE
Update service-search-api.yaml

### PR DESCRIPTION
**Pull request by nkinsler1**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/service-search-api.yaml [hunk 1]
```diff
@@ -117,7 +117,7 @@ paths:
         ## Overview
         Use this endpoint to find information about services provided by NHS organisations.
 
-        For more information on how to use search identifiers with this endpoint, see [DoHS API guide to search identifiers](https://digital.nhs.uk/developer/api-catalogue/directory-of-healthcare-services/guide-to-search-identifiers-and-query-fields).
+        For more information on how to use search identifiers with this endpoint, see [DoHS API guide to search identifiers and service codes](https://digital.nhs.uk//developer/api-catalogue/directory-of-healthcare-services/guide-to-search-identifiers-and-service-codes).
```
Comment (harryhmc) [inline specification/service-search-api.yaml:120]: typo - extra slash in `https://digital.nhs.uk//developer`

File: specification/service-search-api.yaml [hunk 2]
```diff
@@ -310,7 +310,7 @@ paths:
         ## Overview
         Use this endpoint to search for information about services provided by NHS organisations, listed in the service search on the NHS website, filtered by postcode or place name.
 
-        For more information on how to use search identifiers with this endpoint, see [DoHS API guide to search identifiers](https://digital.nhs.uk/developer/api-catalogue/directory-of-healthcare-services/guide-to-search-identifiers-and-query-fields).
+        For more information on how to use search identifiers with this endpoint, see [DoHS API guide to search identifiers and service codes](https://digital.nhs.uk//developer/api-catalogue/directory-of-healthcare-services/guide-to-search-identifiers-and-service-codes).
```
Comment (harryhmc) [inline specification/service-search-api.yaml:313]: same as above

File: specification/service-search-api.yaml [hunk 3]
```diff
@@ -386,7 +386,7 @@ paths:
         ## Overview
         Use this endpoint to get, filter or search the organisation types.
 
-        For more information on how to use search identifiers with this endpoint, see [DoHS API guide to search identifiers and query fields](https://digital.nhs.uk/developer/api-catalogue/directory-of-healthcare-services/guide-to-search-identifiers-and-query-fields).
+        For more information on how to use search identifiers with this endpoint, see [DoHS API guide to search identifiers and service codes](https://digital.nhs.uk//developer/api-catalogue/directory-of-healthcare-services/guide-to-search-identifiers-and-service-codes).
```
Comment (harryhmc) [inline specification/service-search-api.yaml:389]: same as above

**Review by harryhmc — CHANGES_REQUESTED** [review]
pls fix typos in links but otherwise LGTM

**Review by mick-sch — APPROVED** [review]
Fixed 3 URL typos & LGTM now.

**Review by mick-sch — APPROVED** [review]
LGTM

**Review by harryhmc — APPROVED** [review]

**Pull request merged**